src/Rings/MPolyMap/MPolyRing.jl

```diff
@@ -59,7 +59,7 @@ defined by
   x -> y
   y -> x
 with map on coefficients
-#1
+  #1
```

Comment:
@simonbrandhorst do you have an idea what's happening here? The change is possibly due to https://github.com/Nemocas/AbstractAlgebra.jl/pull/1436, but the printing was weird before as well.

Reply:
Its an anonymous function